fusionfrontend/src/app/factory/components/content/asset-card/asset-card.component.scss

```diff
@@ -16,59 +16,100 @@
 @import 'base/typography';
 @import "abstract/variables";
 
-.table-wrapper {
-  min-height: 500px;
-}
-
-.label {
+.asset-card {
```

Comment:
Bei mir am Bildschirm werden aktuell nur 3 Cards nebeneinander angezeigt, die Vierte wird in die zweite Reihe geschoben, wodurch rechts sehr viel Platz bleibt. Liegt aber evtl auch am Browser.